book/content/reproducible_environments/09/case-study-branchLSTM.md

```diff
@@ -0,0 +1,110 @@
+# Case study: branchLSTM
+
+_This project was part of the Alan Turing Institute's Reproducible Research Champions programme, led by Kirstie Whitaker and Martin O'Reilly.
```

Comment:
It might be great to add links to these folk's digital profiles - maybe Github or ResearchGate?

Reply:
Good call - I've added links to everyone's ORCID profiles as we've all got them, and those pages have links out to everyone's personal websites (except for me, but that's a good push to go update my page!).